packages/dds/sequence/src/test/intervalRevertibles.fuzz.spec.ts

```diff
@@ -0,0 +1,228 @@
+
+		assert(isRevertibleSharedString(channel));
+		collection.on("addInterval", (interval, local, op) => {
+			if (local && !channel.isCurrentRevert) {
+				appendAddIntervalToRevertibles(interval, channel.revertibles);
+			}
+		});
+		// Note: delete and change interval edits are disabled for now, and will be reenabled
+		// once bugs AB#4544 and AB#4543 (respectively) are resolved.
+
+		collection.on("deleteInterval", (interval, local, op) => {
+			if (local && !channel.isCurrentRevert) {
+				appendDeleteIntervalToRevertibles(channel, interval, channel.revertibles);
+			}
+		});
+		collection.on("changeInterval", (interval, previousInterval, local, op) => {
+			if (local && !channel.isCurrentRevert) {
+				appendChangeIntervalToRevertibles(
+					channel,
+					interval,
+					previousInterval,
+					channel.revertibles,
+				);
+			}
+		});
+		collection.on("propertyChanged", (interval, propertyDeltas, local, op) => {
+			if (local && !channel.isCurrentRevert) {
+				appendIntervalPropertyChangedToRevertibles(
+					interval,
+					propertyDeltas,
+					channel.revertibles,
+				);
+			}
+		});
+		channel.on("sequenceDelta", (op) => {
+			if (op.isLocal && !channel.isCurrentRevert) {
+				appendSharedStringDeltaToRevertibles(channel, op, channel.revertibles);
+			}
+		});
+	});
+});
+
+const intervalTestOptions: Partial<DDSFuzzSuiteOptions> = {
+	validationStrategy: { type: "fixedInterval", interval: 10 },
+	reconnectProbability: 0,
+	numberOfClients: 3,
+	clientJoinOptions: {
+		maxNumberOfClients: 6,
+		clientAddProbability: 0,
+	},
+	// Once the bugs are resolved, the test count will go back to being set at 100.
+	defaultTestCount: 10,
+	// Uncomment this line to replay a specific seed from its failure file:
+	// replay: 0,
+	saveFailures: { directory: path.join(__dirname, "../../src/test/results") },
+	parseOperations: (serialized: string) => {
+		const operations: Operation[] = JSON.parse(serialized);
+		// Replace this value with some other interval ID and uncomment to filter replay of the test
+		// suite to only include interval operations with this ID.
+		// const filterIntervalId = "00000000-0000-0000-0000-000000000000";
+		// if (filterIntervalId) {
+		// 	return operations.filter((entry) =>
+		// 		[undefined, filterIntervalId].includes((entry as any).id),
+		// 	);
+		// }
+		return operations;
+	},
+};
+
+const optionsWithEmitter: Partial<DDSFuzzSuiteOptions> = {
+	...intervalTestOptions,
+	emitter,
+};
+
+type ClientOpState = FuzzTestState;
+function operationGenerator(
+	optionsParam: OperationGenerationConfig,
+): Generator<RevertOperation, ClientOpState> {
+	async function revertSharedStringRevertibles(
+		state: ClientOpState,
+	): Promise<RevertSharedStringRevertibles> {
+		assert(isRevertibleSharedString(state.channel));
+		return {
+			type: "revertSharedStringRevertibles",
+			// grab a random number of edits to revert
+			editsToRevert: state.random.integer(0, state.channel.revertibles.length),
```

Comment:
```suggestion
			editsToRevert: state.random.integer(1, state.channel.revertibles.length),
```

safe since you already use `hasRevertibles` as an acceptance criterion.